pkg/apis/messaging/v1beta1/kafka_channel_types.go

```diff
@@ -67,6 +67,10 @@ type KafkaChannelSpec struct {
 	// ReplicationFactor is the replication factor of a Kafka topic. By default, it is set to 1.
 	ReplicationFactor int16 `json:"replicationFactor"`
 
+	// RetentionMillis is the retention Time period (in millisecond) of a Kafka topic.
+	// +optional
+	RetentionMillis int64 `json:"retentionMillis,omitempty"`
+
```

Comment:
Did you re-run `update-codegen.sh` ?    Should there be other additions like validate not less than 0 in `kafka_channel_validation.go` ?